stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -206,4 +206,50 @@ interface StripeIntent : StripeModel {
             }
         }
     }
+
+    companion object {
+        internal sealed class NextActionData : StripeModel {
```

Comment:
Precedent is to put nested classes under the class itself, not as a companion object.

Reply:
Yeah, I put it here just so that I could keep it internal, but then I realized I was going to have to make it public anyway so I'll just do that here